sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/TextDocumentInput.java

```diff
@@ -73,9 +73,12 @@ public String getLanguage() {
      * for Spanish etc. If not set, use "en" for English as default.
      *
      * @param language Optional. This is the 2 letter ISO 639-1 representation of a language.
+     *
+     * @return The object {@link TextDocumentInput} itself.
      */
-    public void setLanguage(String language) {
+    public TextDocumentInput setLanguage(String language) {
```

Comment:
i'm not sure i understand the context on just making setLanguage a setter and not adding it to a separate constructor like you used to have it. Is there a reason why you can't add this constructor back like it is in master: https://github.com/Azure/azure-sdk-for-java/blob/fcdaff1ebc84a6f660f94c54038c2c34fef76e0c/sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/TextDocumentInput.java#L50

I really don't think it's a good idea to force users to call TextDocumentInput.setLanguage("en") for the most basic version of TextDocumentInput (you're doing this in your tests). What I think you should do is have two constructors exactly how it is in master. Is there a reason why you're changing this code?

Reply:
the only time constructor needed is when the parameter is required. But since `language` is optional. it should not go to constructor. Let's say maybe in the future, more parameter will be added to this class, we will not add another constructor if the value is not required.